docs/openshift-version-in-the-report.md

```diff
@@ -0,0 +1,73 @@
+# OpenShift Version in the Report
+
+The chart-verifier tool adds the OpenShift version where chart-testing has run
+in the report.  The information is available as a metadata at this path:
+`.metadata.tool.certifiedOpenShiftVersions`.  Here is an example showing only
+the relevant details:
```

Comment:
Please hard wrap this into one single line.

Reply:
Text wrapping at 80 columns is good for PR review (diff) and writing in a text editor from a terminal.

Reply:
What is the benefit of a single line?  I find multi-line version is easier to work with.

Reply:
RH writing guidelines: Hard wrapping makes the code nice and tidy, as there are some editors that do not wrap lines when you open a new file. It allows you to concentrate on the structure of your document. Good indentation and good wrapping make it more readable by most of users. Please do take it as a habit :).